zhaquirks/tuya/ts0601_motion.py

```diff
@@ -66,6 +82,94 @@ class TuyaRelativeHumidity(RelativeHumidity, TuyaLocalCluster):
     """Tuya local RelativeHumidity cluster."""
 
 
+class TuyaMmwRadarSensitivity(TuyaAttributesCluster, AnalogOutput):
+    """AnalogOutput cluster for sensitivity."""
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        super().__init__(*args, **kwargs)
+        self._update_attribute(self.attributes_by_name["description"].id, "sensitivity")
+        self._update_attribute(self.attributes_by_name["min_present_value"].id, 1)
+        self._update_attribute(self.attributes_by_name["max_present_value"].id, 9)
+        self._update_attribute(self.attributes_by_name["resolution"].id, 1)
```

Comment:
With recent-ish zigpy updates, we should be able to use `AnalogOutput.AttributeDefs.description.id` here (`self.AttributeDefs.description.id` should also work, although I think I prefer the more "explicit" version).

I think it's nicer to use `AttributeDefs` everywhere.

Reply:
Also, can't we just use `_CONSTANT_ATTRIBUTES ` for all of this?
Example:
https://github.com/zigpy/zha-device-handlers/blob/7385465bd6f04f295893fb20d053433011183935/zhaquirks/terncy/cl001.py#L30-L34